core-bundle/src/Resources/contao/library/Contao/Email.php

```diff
@@ -120,13 +124,13 @@ class Email
 	/**
 	 * Instantiate the object and load the mailer framework
 	 *
-	 * @param \Swift_Mailer|null $objMailer
+	 * @param MailerInterface|null $objMailer
 	 */
-	public function __construct(\Swift_Mailer $objMailer = null)
+	public function __construct(MailerInterface $objMailer = null)
```

Comment:
That's a BC break. I wonder if we could make the class deal with both?

Reply:
True. Yeah it would be no problem to make the class deal with both. It's just very cumbersome, because we will need a lot of this then:

```php
if ($this->objMailer instanceof MailerInterface) {
    $this->objMessage->to(…);
} else {
    $this->objMessage->setTo(…);
}
```

Reply:
If we want to preserve BC, we will have to implement it this way I guess.

Reply:
PR updated with `\Swift_Mailer`/`\Swift_Message` fallback.